pcdsdevices/interface.py

```diff
@@ -1322,7 +1330,10 @@ def state(self):
             if method_name.startswith('wm_'):
                 state_name = method_name.replace('wm_', '', 1)
                 wm_state = getattr(device, method_name)
-                diff = abs(wm_state())
+                state_val = wm_state()
+                if not isinstance(state_val, (int, float)):
```

Comment:
We don't ever get numpy floats in these, right?

Reply:
Unsure, but it's probably prudent to future-proof ourselves.  I thought about this for half a second then forgot about it.  `numbers.Number` should work.

```python
In [5]: for n in [1, 1.2, np.int32(3.3), np.float64(3.2)]:
   ...:     print(isinstance(n, numbers.Number))
   ...: 
True
True
True
True
```